packages/edit-site/src/components/page-pages/index.js

```diff
@@ -117,6 +121,18 @@ export default function PagePages() {
 					postStatuses[ page.status ] ?? page.status,
 				enableSorting: false,
 			},
+			{
+				header: 'Date',
+				id: 'date',
+				cell: ( props ) => {
```

Comment:
Following up [this rationale](https://github.com/WordPress/gutenberg/pull/55196#issuecomment-1754970796), what do you think of doing this instead (pseudo-code):

```js
accessorFn: ( page ) => {
    return dateI18n(
        getSettings().formats.datetimeAbbreviated,
        getDate( props.row.original.date )
    );
}
cell: ( props ) => { return <time>{props.row.getValue()}</time> }
```

Reply:
I think we need to redo `accessorFn` and `cell` to be less tied to the API of tanstack at some point, and yeah we need to agree on a given way of retrieving the fields data.

Reply:
I'd echo Riad on this. We need to start forming the API that will map internally to tanstack.